src/main/resources/db/migration/V1_54__appointment.sql

```diff
@@ -0,0 +1,44 @@
+-- CREATE TYPE attended AS ENUM ('YES','LATE', 'NO');
+
+create table appointment(
+    id uuid not null,
+    appointment_time timestamp with time zone null,
+    duration_in_minutes int null,
+    created_at timestamp with time zone not null,
+    created_by_id text not null,
+
+    attended attended,
+    additional_attendance_information text,
+    attendance_submitted_at timestamp with time zone,
+
+    attendance_behaviour text,
+    notifyppof_attendance_behaviour boolean,
+    attendance_behaviour_submitted_at timestamp with time zone,
+
+    session_feedback_submitted_at timestamp with time zone,
+    delius_appointment_id bigint,
+
+    constraint pk_appt_id primary key (id),
+    constraint fk_appt_to_created_by foreign key (created_by_id) references auth_user
+);
```

Comment:
can we copy over existing appointments into the new table before dropping the old columns?